plugin.yml

```diff
@@ -1,6 +1,8 @@
 name: LaunchableTNT
-api: 3.0.0
+api: 4.0.0
 main: Vecnavium\LaunchableTNT\LaunchableTNT
-version: 1.0.1
-author: Vecnavium
+version: 1.0.2
+author: 
```

Comment:
Please revert this change and leave it as only Vecnavium as it's only supposed to display the author and not contributors